txpool/pool.go

```diff
@@ -1723,7 +1725,7 @@ func (p *TxPool) fromDB(ctx context.Context, tx kv.Tx, coreTx kv.Tx) error {
 		txs.Resize(uint(i + 1))
 		txs.Txs[i] = txn
 		txs.IsLocal[i] = isLocalTx
-		copy(txs.Senders.At(i), addr)
+		copy(txs.Senders.At(i), addr[:])
```

Comment:
I think this line is broken - because `At` now returns copy of data instead of reference on underlying buffer

Reply:
I think this line is not broken, method _At_ still returns a pointer to array, only SliceHeader is copying (len, cap and pointer to array)